theforeman.org/yaml/jobs/test_plugin.yaml

```diff
@@ -29,6 +29,7 @@
           block: true
     publishers:
       - ircbot_freenode
+      - email_list
```

Comment:
There's no need to modify all jobs. You can override the publishers in the bootdisk plugin config.

Reply:
Sure but woudn't it be more consistent to send all failures there? I think our community could benefit from this, unless it creates too much noise. These are executed weekly, we should be fine.

Reply:
I think it creates too much noise and would prefer to start with one. I don't want to end up with a board that has so many things there that nobody looks anymore.

Reply:
Let's enable that just for discovery and see how it works.